### ISSUE TITLE
Update image tags to represent versions of each application

### ISSUE THREAD

**Issue by dskaggs** [LodestoneHQ/lodestone#104]
Once images have been published to a DockerHub organization, update the tags of each to be a version number rather than the name of the container.

**Comment by adam-stanek** [comment]
It is not only about splitting the images / using version tags properly, but it would be also good to split the build as well. I have been tinkering with the lodestone-processor repo and it sucks that you cannot easily build image out of it. Instead the docker image is built in the lodestone repo but only using artifacts already published to the repository (https://github.com/LodestoneHQ/lodestone/blob/master/processor/Dockerfile#L9). This effectively closes your project to any contributors because you have no easy way of building image out of fork / merge request.

Instead of this each repo should be responsible for building itself. There is no real benefit of this added complexity.

**Comment by dskaggs** [comment]
@adam-stanek Would you mind creating an issue for the changes you'd like to see in the build processes for each component?

**Comment by adam-stanek** [comment]
Added #111 and #112

**Comment by dskaggs** [comment]
> Added #111 and #112

Thank you! Looking over the PR now. I'll have some questions to help my understanding of how the changes fit into the overall docker-compose but I'll put those over on those tickets.

**Comment by adam-stanek** [comment]
I looked at the built images on docker hub and noticed you are only building `latest` tags. Do you think it would be possible to also tag individual releases? I think it is crucial for production use when you want to prevent potential breaking changes in cloud deployments. The deployment should always be repeatable and transferable between nodes (you need stable images).

**Comment by dskaggs** [comment]
> Do you think it would be possible to also tag individual releases

Yes definitely. They're all set up to build a version tag any time we actually release a version in GitHub. I just haven't release anything new yet as I was waiting for things to get more stable. You can see there's a `latest` and a `v0.2.1` tag for the main lodestone repo (https://hub.docker.com/r/lodestonehq/lodestone/tags?page=1&ordering=last_updated). The other repos will do that as well when a new release is tagged.

**Comment by adam-stanek** [comment]
Good to know 👍
